src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -67,6 +67,7 @@ class Response
     const HTTP_PRECONDITION_REQUIRED = 428;                                       // RFC6585
     const HTTP_TOO_MANY_REQUESTS = 429;                                           // RFC6585
     const HTTP_REQUEST_HEADER_FIELDS_TOO_LARGE = 431;                             // RFC6585
+    const HTTP_UNAVAILABLE_FOR_LEGAL_REASONS = 451;
```

Comment:
It's the reason I didn't make a PR yet yesterday, it was still a draft so I could not attach an RFC.

Is the master the right branch? I think this could easily be addd to 2.3, 2.7, 2.8 and 3.0 as well.